frontend/src/model/beta/releases/ReleaseForm.tsx

```diff
@@ -54,6 +58,8 @@ export default function ReleaseForm({
   isEdit = false,
 }: ReleaseFormProps) {
   const isReadOnly = useMemo(() => editable && !isEdit, [editable, isEdit])
+  const [latestRelease, setLatestRelease] = useState<string>('')
```

Comment:
This can be simplified to `useState('')`. It'll automatically type is as a string as you've initialised it as so